src/routes/welcomeRoute.js

```diff
@@ -21,9 +20,4 @@ const router = Router();
 
 router.get('/', (_req, res) => Response.success(res, 200, 'Welcome to barefoot nomand'));
 
```

Comment:
better use status code names

Reply:
Yeah, It's resolved @Mucyo-Aime-Christian